snippets/mco-mcn-ocl-example.adoc

```diff
@@ -0,0 +1,37 @@
+// Text snippet included in the following modules:
+//
+// * modules/coreos-layering-configuring-on
+// * modules/checking-mco-node-status
+
+.Example machine config node output
+[source,terminal]
+----
+Name:         ip-10-0-14-86.us-west-1.compute.internal
+API Version:  machineconfiguration.openshift.io/v1
+Kind:         MachineConfigNode
+# ...
+Spec:
+  Config Image:
+    Desired Image:  image-registry.openshift-image-registry.svc:5000/openshift-machine-config-operator/ocb-image@sha256:b485378fd8f7963ed74f14ce64f4f1e511e1601d49302b3046b1b78a83f539e3 <1>
+  Config Version:
+    Desired:  rendered-worker-d63c7736923b60b8b82492ae9a1eef40
+  Node:
+    Name:  ip-10-0-14-86.us-west-1.compute.internal
+  Pool:
+    Name:  worker
+# ...
+Status:
+  Conditions:
+# ...
+    Message:               Action during update to image-registry.openshift-image-registry.svc:5000/openshift-machine-config-operator/ocb-image@sha256:b485378fd8f7963ed74f14ce64f4f1e511e1601d49302b3046b1b78a83f539e3: Successfully pulled OS image image-registry.openshift-image-registry.svc:5000/openshift-machine-config-operator/ocb-image@sha256:b485378fd8f7963ed74f14ce64f4f1e511e1601d49302b3046b1b78a83f539e3 from registry
+    Reason:                ImagePulledFromRegistry
+    Status:                False
+    Type:                  ImagePulledFromRegistry
+# ...
+  Config Image:
+    Current Image:  image-registry.openshift-image-registry.svc:5000/openshift-machine-config-operator/ocb-image@sha256:b485378fd8f7963ed74f14ce64f4f1e511e1601d49302b3046b1b78a83f539e3
+    Desired Image:  image-registry.openshift-image-registry.svc:5000/openshift-machine-config-operator/ocb-image@sha256:b485378fd8f7963ed74f14ce64f4f1e511e1601d49302b3046b1b78a83f539e3
+# ...
+----
+<1> Digested image pull spec for the new custom layered image.
```

Comment:
I _think_ we need to be doing one of these now for callouts: https://redhat-documentation.github.io/supplementary-style-guide/#explain-commands-variables-in-code-blocks